docs/content/en/docs/intro-klt.md

```diff
@@ -61,6 +61,15 @@ It supports the following steps:
 
 All of these things can be executed for a workload or for a KeptnApp.
```

Comment:
Make `KeptnApp` a link to https://lifecycle.keptn.sh/docs/yaml-crd-ref/app/ , which has contents in the `main` branch although that has not yet been published.  But it shows in your local branch, the netlify build, and I think also if you select the `development` release from the Release drop-down on the published doc site